app/src/ui/v_configuration.py

```diff
@@ -1,9 +1,9 @@
 import streamlit as st
-from snowflake.snowpark.context import get_active_session
 from first_time_setup import get_is_first_time_setup_dismissed, \
     render as render_first_time_setup
+from snowflake.snowpark import Session
 
-session = get_active_session()
+session = Session.builder.getOrCreate()
 
 def check_warning_task_enabled():
     is_warning_task_enabled = session.sql(f"""
```

Comment:
why are these reading the global var instead of accepting the session directly, as above?

Reply:
Does Snowpark have any guidance we should align with?

Reply:
I checked with them - no specific guidance. I don't like using globals either. I will refactor that.